x-pack/platform/packages/private/security/ui_components/src/kibana_privilege_table/components/feature_table_cell.tsx

```diff
@@ -37,6 +37,14 @@ export const FeatureTableCell = ({ feature, className }: Props) => {
     );
   }
 
+  const showFeatureDescription = React.useMemo(
+    () =>
+      feature.description &&
+      feature.description.length > 0 &&
+      feature.getSubFeaturePrivileges().length > 0,
```

Comment:
This logic is to hide description for feature added in https://github.com/elastic/kibana/pull/147709 and updated in https://github.com/elastic/kibana/pull/192273

Reply:
You are right @azasypkin. I've reverted the change.

Reply:
Great, thanks 👍